.github/workflows/pg-migtests.yml

```diff
@@ -160,22 +160,14 @@ jobs:
       - name: "TEST: pg-hasura-ecommerce"
         run: migtests/scripts/run-test.sh pg/hasura-ecommerce
 
-      # Placeholder for now so that a basic test can run
-      - name: Create the live migration user
-        run: |
-          psql "postgresql://postgres:secret@127.0.0.1:5432/postgres" -c "ALTER ROLE ybvoyager WITH SUPERUSER;"
-
-      - name: "TEST: pg-basic-live-migration-test"
-        run: migtests/scripts/live-migration-run-test.sh pg/basic-live-test
+      - name: "TEST: pg-basic-non-public-live-migration-test"
+        run: migtests/scripts/live-migration-run-test.sh pg/basic-non-public-live-test
 
-      - name: "TEST: pg-basic-live-migration-fall-forward-test"
-        run: migtests/scripts/live-migration-fallf-run-test.sh pg/basic-live-test
+      - name: "TEST: pg-basic-non-public-fall-forward-test"
+        run: migtests/scripts/live-migration-fallf-run-test.sh pg/basic-non-public-live-test
         if: matrix.version != '2.16.7.0-b59'
       
-      - name: "TEST: pg-datatypes-live-migration-test"
-        run: migtests/scripts/live-migration-run-test.sh pg/datatypes
-        if: matrix.version != '2.16.7.0-b59' && matrix.BETA_FAST_DATA_EXPORT == 0 #TODO: fix its validation
+      - name: "TEST: pg-basic-non-public-fall-back-test"
+        run: migtests/scripts/live-migration-fallb-run-test.sh pg/basic-non-public-live-test
+        if: matrix.version != '2.16.7.0-b59'
```

Comment:
Why are all non-public? at least one public live migration test can be there right?